.mvn/jvm.config

```diff
@@ -0,0 +1 @@
+-Xmx512m -XX:+HeapDumpOnOutOfMemoryError -XX:+TieredCompilation -XX:TieredStopAtLevel=1
```

Comment:
I am assuming that this was just copied from elsewhere and no investigation was done on the max heap?